style/javascript/README.md

```diff
@@ -1,8 +1,9 @@
-JavaScript
-==========
+# JavaScript
 
 [Sample](sample.js)
 
+* Use [ESlint] and [Prettier] for auto-formatting and auto-fixing
```

Comment:
Do ESLint and Prettier eliminate the needs for the other guidelines here? (trailing commas, `===`, etc)

Reply:
My thought was that ESLint and Prettier enforce the style guide, but that it was still worth listing the things we care most about directly in the guide. Alternately we could treat the ESLint and Prettier configs as the definitive source, but I find them to be less digestable for someone coming to the style guides and wanting to quickly read over all the guidelines.

Reply:
Devil's advocate, @composerinteralia authored https://github.com/thoughtbot/guides/pull/606 today which removes the ruby guidelines in favor of just using and referencing [standard](https://github.com/testdouble/standard).

I like the simplicity of that, and not having to maintain guidelines in both a configuration file and a textual guide.

Curious what others think as well.

Reply:
Based on https://github.com/thoughtbot/guides/pull/606#issuecomment-643308916 and the discussion in https://github.com/thoughtbot/guides/issues/610 it looks like we might be trending towards removing guidelines that we can't automate and don't feel strongly about. Linking those from here so we can continue the conversation when we condense the guides.